packages/cloud-cognitive/src/components/UserProfileImage/_colorMap.scss

```diff
@@ -1,3 +1,10 @@
+//
```

Comment:
All SCSS files should use kebab case. Perhaps change this file to `_color-map.scss`?